### PR TITLE
11 create content section community

### PR DESCRIPTION
**Pull request by MarwaKHTanani**
add remaining photos in the assets file and elements. PS: photos dose not been added yet inside the elements

### CODE REVIEW

File: community/community.html [hunk 1]
```diff
@@ -4,10 +4,82 @@
...
+              <a href="#" class="item">Popular</a>
+              <a href="#" class="item">Commented</a>
+              <a href="#" class="item">Preparation Time</a>
+              <a href="#" class="item">Rating</a>
+            </nav>
+        </section>
+
+        <section id="Relevancy" class="content">
+            <div class="categories">
+               <a href=""> <img  src="" alt="food"></a>
```
Comment (mo7mad4) [inline community/community.html:30]: ### put the picture here

File: community/community.html [hunk 2]
```diff
@@ -4,10 +4,82 @@
...
+                <h1>Resep Sayur  Asem Betawi </h1>
+                <div class="people">
+                    <img src="" alt="girl">
+                    <h3>Susanti</h3>
+                    <span>105 Reviewers</span>
+                </div>
+            </div>
+            <div class="categories">
+                <img src="" alt="food">
```
Comment (mo7mad4) [inline community/community.html:39]: ### put the picture here

**Review by mo7mad4 — DISMISSED** [review]

**Review by shamskhodary — APPROVED** [review]

File: community/community.css [hunk 3]
```diff
@@ -0,0 +1,134 @@
...
+}
+.people img{
+    width: 50px;
+    height: 50px;
+}
+.people{
+    display: inline-flexbox;
+    padding-bottom: 8px;
+}
+.people h3 , span{
```
Comment (shamskhodary) [inline community/community.css:125]: It would be nice if you use the design fonts, which are Recoleta and Cera pro. Otherwise, You rocked it Marwa ^.^

Comment (shamskhodary) [reply community/community.css:125]: @import url("http://fonts.cdnfonts.com/css/recoleta");
@import url("http://fonts.cdnfonts.com/css/cera-pro");

Add them in the CSS file and change each font family that matches the design text fonts.

**Review by ShbairAli — APPROVED** [review]

**Review by mo7mad4 — APPROVED** [review]

**Review by shamskhodary — DISMISSED** [review]

**Review by mo7mad4 — APPROVED** [review]

**Review by shamskhodary — APPROVED** [review]

**Review by ShbairAli — APPROVED** [review]

**Pull request merged**